### PR TITLE
Wizard recipe: JSON_C-v0.15.0

### PR DESCRIPTION
**Pull request by bramtayl**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: JSON_C
* Version: v0.15.0

@staticfloat please review and merge.

### CODE REVIEW

File: J/JSON_C/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libjson-c", :JSON_C)
```
Comment (giordano) [inline J/JSON_C/build_tarballs.jl:28]: Out of curiosity, why not
```suggestion
    LibraryProduct("libjson-c", :libjson_c)
```
?

Comment (bramtayl) [reply J/JSON_C/build_tarballs.jl:28]: It was the little they listed on their readme https://github.com/json-c/json-c. I could change it to libjson-c. What does lib mean?

Comment (giordano) [reply J/JSON_C/build_tarballs.jl:28]: "lib" stands for "library" :slightly_smiling_face:

Comment (bramtayl) [reply J/JSON_C/build_tarballs.jl:28]: Ok, I'll change it

**Pull request closed without merging**